### PR TITLE
Added a tab-bar to the submission page

### PR DESCRIPTION
**Pull request by hepcat72**
## Summary Change Description

This ONLY adds the tab-bar.  The tabs are clickable to change pages, but this PR is mainly for aesthetics and CSS review.

Details:

- Added a CSS file for the customized css based on the bootstrap code.
- Added DataValidationView.get_context_data to be able to pass the page as a URL parameter.
- Added HTML to the submission template.

Screenshot:

tabbar

Much of this work was informed by the following documentation:

https://getbootstrap.com/docs/4.1/components/navs/

## Affected Issues/Pull Requests

- Partially addresses #1093
- Merges into PR #1113
- Next PR: #1134

## Review Notes

See comments in-line.

## Checklist

This pull request will be merged once the following requirements are met.  The
author and/or reviewers should uncheck any unmet requirements:

- Review requirements
  - Minimum approvals: 1 
  - No changes requested
  - All blocking issues resolved by reviewers
  - Specific reviewers: @__add_username_here__
    
  - Review period: 2 days 
- Associated issue/pull request requirements:
  
  - [x] All requirements in affected issues marked "resolved" are satisfied
  - [x] All required pull requests are merged *(or none)*
- Basic requirements
  
  - [x] [All linters pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting)
  - [x] [All tests pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)
  - [x] All conflicts resolved
- Overhead requirements
  
  - [x] [New/changed method tests implemented/updated *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
  - [ ] [Updated *Unreleased* section of `changelog.md` *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/changelog.md)
  - [x] [Migrations created & committed *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)

### CODE REVIEW

File: DataRepo/templates/DataRepo/validate_submission.html [hunk 1]
```diff
@@ -59,6 +60,20 @@
...
+            <li class="tabbar-item">
+                <a class="tabbar-link {% if page == 'Fill In' %}active{% endif %}" href="{% url 'validate' %}?page=Fill%20In">Fill In</a>
+            </li>
+            <li class="tabbar-item">
+                <a class="tabbar-link {% if page == 'Validate' %}active{% endif %}" href="{% url 'validate' %}?page=Validate">Validate</a>
+            </li>
+            <li class="tabbar-item">
+                <a class="tabbar-link {% if page == 'Submit' %}active{% endif %}" href="{% url 'validate' %}?page=Submit">Submit</a>
+            </li>
+        </ul>
```
Comment (lparsons) [inline DataRepo/templates/DataRepo/validate_submission.html:76]: Beautiful, nice and clean HTML describing the structure.

File: DataRepo/views/upload/validation.py [hunk 2]
```diff
@@ -505,6 +505,11 @@ def set_files(
                 "ambiguous due to common header names."
             )
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["page"] = self.request.GET.get("page", None)
+        return context
+
```
Comment (lparsons) [inline DataRepo/views/upload/validation.py:512]: Q: I'm not clear why this is needed, it doesn't seem to be referenced in this PR. Maybe it's extraneous at this point and can be removed?

**Review by lparsons — APPROVED** [review]
Looks really slick, nice job on the UI stuff!

File: DataRepo/views/upload/validation.py (hunk 2 above)
Comment (hepcat72) [reply DataRepo/views/upload/validation.py:512]: It's referenced in `DataRepo/templates/DataRepo/validate_submission.html`, twice on each of line 65, 68, 71, and 74 (once as the context variable and once setting it as a URL param).

Comment (hepcat72) [reply DataRepo/views/upload/validation.py:512]: It's what makes the tabs clickable.

Comment (lparsons) [reply DataRepo/views/upload/validation.py:512]: Ah, I see, it's a [special Django function that we are overwriting](https://docs.djangoproject.com/en/5.0/topics/class-based-views/generic-display/#adding-extra-context). I didn't see `get_context_data` anywhere and I wasn't sure how it was getting used. 👍🏻

**Pull request merged**